bluez-generated/README.md

```diff
@@ -3,26 +3,39 @@
 [![crates.io page](https://img.shields.io/crates/v/bluez-generated.svg)](https://crates.io/crates/bluez-generated)
 [![docs.rs page](https://docs.rs/bluez-generated/badge.svg)](https://docs.rs/bluez-generated)
 
-This crate contains async bindings for Bluez.
+This crate contains async D-Bus bindings for BlueZ.
 
-Bindings are generated from introspection data, using `dbus-codegen`.
-This means that it is relatively easy to maintain, but it only covers interfaces
-that I have the devices for.
+Bindings are generated from introspection data, using
+[`dbus-codegen`](https://crates.io/crates/dbus-codegen). This means that it is relatively easy to
+maintain, but it only covers interfaces that I have the devices for.
 
 ## Adding Interfaces
 
-If there is an interface that you need which is not generated, it should be
-reasonably straightforward to generate them and send a pull request. See
+If there is an interface that you need which is not generated, it should be reasonably
+straightforward to generate them and send a pull request. See
 [introspect.sh](https://github.com/alsuren/mijia-homie/blob/master/bluez-generated/introspect.sh)
-for details. It's also perfectly reasonable to generate the interfaces you need
-and vendor them into your project.
+for details. It's also perfectly reasonable to generate the interfaces you need and vendor them into
+your project.
 
 ## Future Direction
 
-Only async bindings are generated. Blocking bindings could also be generated,
-but I'm unlikely to use them, so they would need to be contributed by someone
-else.
+Only async bindings are generated. Blocking bindings could also be generated, but I'm unlikely
+to use them, so they would need to be contributed by someone else.
 
-It would be nice to generate some strongly typed bindings around
-`get_managed_objects()` for bulk-fetching properties and `add_match()` for
-subscribing to events.
+It would be nice to generate some strongly typed bindings around `add_match()` for subscribing to
```

Comment:
What does this mean actually?

Reply:
Let's say we have an interface with a `connected` property which is known to be boolean. It would be interesting to generate a `.connected_changes() -> Stream<Item=bool>` that listens for property-changed signals and presents them as a stream.

Reply:
Okay, I've clarified that. Though I think the match would just have to be on the `PropertiesChanged` signal, which would include all properties of the object, so it might make sense to return a stream of the *Properties structs that dbus-codegen now generates.